src/schemas/Polymorphic.js

```diff
@@ -41,10 +43,11 @@ export default class PolymorphicSchema {
   }
 
   denormalizeValue(value, unvisit) {
-    if (!this.isSingleSchema && !value.schema) {
+    const mutableValue = isImmutable(value) ? value.toJS() : value;
```

Comment:
Do we have to convert `toJS()`? My (limited) understanding is that it's expensive. Can't we use `mutableValue.get('schema')`?